pkg/ccl/backupccl/show.go

```diff
@@ -926,6 +940,42 @@ func nullIfZero(i descpb.ID) tree.Datum {
 	return tree.NewDInt(tree.DInt(i))
 }
 
+func showRegions(descriptor catalog.TypeDescriptor, dbname string) string {
+	var regionsStringBuilder strings.Builder
+	if descriptor == nil {
+		return ""
+	}
```

Comment:
aha! in general, don't swallow errors! i'd change this func return signature to `(string, err)` and on the error path, return `"", err`.